packages/hypercontroller/test/jwt.spec.ts

```diff
@@ -34,7 +35,7 @@ class Api {
 
 const app = createServer({
   controllers: [Api],
-  opts: { logging: { logger } },
+  opts: { logging: { logger }, cookieSecret: 'evil-supply' },
```

Comment:
Prefer instead of `evil-supply` something that shows the config value: `auth-cookie-secret`

Reply:
Fixed